cpu/esp32/esp-now/esp_now_netif.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2018 Timo Rothenpieler
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup net_gnrc_netif
```

Comment:
Please add it also to group `cpu_esp32_esp_now`
```
 * @ingroup     cpu_esp32_esp_now
```

Reply:
Fixed